ui/admin/tests/helpers/selectors.js

```diff
@@ -28,6 +30,8 @@ export const PAGE_MESSAGE_LINK =
 export const HREF = (url) => `[href="${url}"]`;
 
 export const SIDEBAR_NAV_LINK = (url) => `[title="General"] a[href="${url}"]`;
```

Comment:
I realized this selector is just for the top level side nav like Orgs and Projects. I will rename this in a separate PR as well.